common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -176,8 +176,7 @@ public boolean getTrustedPeerOnly() {
     }
 
     public Uri getBlockExplorer() {
-        return Uri.parse(prefs.getString(PREFS_KEY_BLOCK_EXPLORER,
-                res.getStringArray(R.array.preferences_block_explorer_values)[0]));
+        return Uri.parse(prefs.getString(PREFS_KEY_BLOCK_EXPLORER, null));
```

Comment:
was this an intended change for this fix?

Reply:
Perhaps because it was because this file was moved:  common/src/main/res/values/values.xml

But, then the getBlockExplorer will return null on new installs and that might make the "View in Explorer" link crash on the Tx Details screens.